Woodog/woodogdata/urls.py

```diff
@@ -2,5 +2,6 @@
 from . import views
```

Comment:
If you edit the `templates/authentication/register.html` (as I've said in my first comment), these changes in the templates `/woodogdata/urls.py` and `/woodogodata/views.py` will not be required.